src/main/java/org/example/C05_P44_Repeating_Elements_In_Array.java

```diff
@@ -0,0 +1,27 @@
+package org.example;
+import java.util.Arrays;
+/******************************************************************************
+ 44.Write a program in java to find the two repeating elements in a given array.
+ Expected Output :
+ The given array is : 2 7 4 7 8 3 4
+ The repeating elements are: 7 0
```

Comment:
0 is not a repeating element.

Reply:
ok. I changed... description of the problem was not good. the repeating element must be 4 and 7.. and test is checking correctly